src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
Combine it with `transition` and set each property for which it is used
![image](https://user-images.githubusercontent.com/70009205/233127920-0e910fa4-7672-4d6e-987f-8600a96fe5f9.png)